src/mpp/Dec.hpp

```diff
@@ -57,6 +57,10 @@ constexpr bool is_any_putable_v =
 	tnt::is_emplacable_v<T> || tnt::is_back_emplacable_v<T> ||
 	tnt::is_back_pushable_v<T> || tnt::is_insertable_v<T>;
 
+template <class T, class BUF>
+constexpr bool is_raw_decoded_v =
+	is_wrapped_raw_v<T> || tnt::is_pairish_of_v<unwrap_t<T>, BUF, BUF>;
```

Comment:
Nit: why `BUF`, shouldn't it be `ITER`?

Reply:
We use `BUF` in decoder.
In fact, we use `mpp::decode` with iterator only. But decoder is designed to use any object with `read` and `get`, which can be a buffer.
But the check implicitly implies, that `BUF` is an iterator when `pair<BUF, BUF>` is passed - it would be really strange to pass a pair of heavy buffers.

I think we should introduce traits for iterators and buffers later and use them here to check if it is actually a pair of _iterators_, but raw decoding is a blocker for SQL, so I would leave it as it is and return to it later.